examples/hello-world/tutorial.md

```diff
@@ -131,33 +131,36 @@ In the starter template project, our host driver program creates an executor env
 
 ## Step 5 (Host): Share two values with the guest
 
-In this step, we'll be continuing to modify `hello-world/src/main.rs`.
-Let's start by picking some aesthetically pleasing primes:
+In this step, we'll be continuing to modify `hello-world/host/src/main.rs`.
+Let's start by picking some aesthetically pleasing primes such as `17` and `23`.
+They will be added to the top of our `main` function:
 ```
 fn main() {
     let a: u64 = 17;
     let b: u64 = 23;
-}
+    //...snip...
```

Comment:
We need to leave the code snippets testable (or add `no_compile` if that's not practical)
```suggestion
    // ...snip...
}
```

Reply:
should be good in latest commit